CMSgov/onemac
# OY2 23421: PROD Hot Fix for TE Submissions Issues

Story: https://qmacbis.atlassian.net/browse/OY2-23421
Endpoint: See github-actions bot comment

### Details
Temporary Extensions were validating the parent package on OneMAC packages only.  Need to also dive into SEA Tool packages to validate parents.

NOTE: 1915b waiver renewals and amendments also need to be able to validate parents from SEA Tool only.

### Changes
- validateParentOfAny searches the SEATool topic items if the ID did not match a OneMAC package
- Used the PlanType+ActionType as the componentType
- Used the StatusDescription as the currentStatus
- Updated the configs for TEs to allow SEATool types of 1915(b)New, 1915(b)Renew, 1915(c)New and 1915(c)Renew
- Updated the configs for 1915b Renewals to allow SEATool types of 1915(b)New and 1915(b)Renew
- Updated the configs for 1915b Amendments to allow SEATool types of 1915(b)New and 1915(b)Renew
- Added a territory check on the parent ID field on the form.
- Updated one-seed to include test SEATool events
- Updated resetData to reset those new events as snapshots

### Implementation Notes
- This is a quick fix to get the forms working properly while we work on a refactor that will also get the forms working, but will do it in a manner consistent with our data design.

### Test Plan
1. Login as a submitting user (MD has the test records)
2. Click "New Submission->Waiver Action->Request Temporary Extension"
3. In the Approved Initial or Renewal Waiver Number field...
a. Try a territory not allowed by user to see state access message for the parent ID
b. Valid IDs are: MD-0645.R00.00, MD-40198.R02, MD-01.000, MD.20230,
c. Invalid IDs are: MD-9996, MD-0645.R01, MD-0265.R04.02, MD-40198.R02.01, MD-9995, MD-02.R00.M04, MD-21-0999-SID
4. Using a valid ID, create a Temporary Extension
5. Verify it submits correctly
6. Verify TE is on the dashboard
7. Click "New Submission->Waiver Action->1915b Waiver Renewal"
3. In the Existing Waiver Number to Renew field...
a. Try a territory not allowed by user to see state access message for the parent ID
b. Valid IDs are: MD-01.000, MD.20230,
c. Invalid IDs are: MD-0645.R00.00, MD-40198.R02, MD-9996, MD-0645.R01, MD-0265.R04.02, MD-40198.R02.01, MD-9995, MD-02.R00.M04, MD-21-0999-SID
4. Using a valid ID, create a Waiver Renewal
5. Verify it submits correctly
6. Verify Renewal is on the dashboard
7. Click "New Submission->Waiver Action->1915b Waiver Amendment"
3. In the Existing Waiver Number to Amend field...
a. Try a territory not allowed by user to see state access message for the parent ID
b. Valid IDs are: MD-01.000, MD.20230,
c. Invalid IDs are: MD-0645.R00.00, MD-40198.R02, MD-9996, MD-0645.R01, MD-0265.R04.02, MD-40198.R02.01, MD-9995, MD-02.R00.M04, MD-21-0999-SID
4. Using a valid ID, create a Waiver Amendment
5. Verify it submits correctly
6. Verify Amendment is on the dashboard
